tools/Newtonsoft.Json.xml

```diff
@@ -0,0 +1,10760 @@
+<?xml version="1.0"?>
```

Comment:
Also, we should clean up the commits.  No reason this shouldn't be in a single commit.